### PR TITLE
Security_onion parser unit test

### PR DESCRIPTION
**Pull request by tlaurion**
# Description

Quick unit test to validate if there was format change in the way SecurityOnion was logging things up.
No change. Now we have a unit test!
# Impacts

none
# Delete branch after merge

YES

### CODE REVIEW

File: t/unittest/detect/parser/securityonion.t [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#This test will running last
+use Test::NoWarnings;
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2015 Inverse inc.
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t]: 2016

File: t/unittest/detect/parser/securityonion.t [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+=head1 DESCRIPTION
+
+example pf test script
+
+=cut
+
+use strict;
+use warnings;
+#
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t]: remove

File: t/unittest/detect/parser/securityonion.t [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+BEGIN {
+    #include test libs
+    use lib qw(/usr/local/pf/t);
+    #Module for overriding configuration paths
+    use setup_test_config;
+}
+
+use_ok('pf::factory::detect::parser');
+
+#my $alert = 'Oct 28 13:37:42 poulichefencer sguil_alert: 13:37:42 pid(3403)  Alert Received: 0 2 misc-attack securityonion1-eth1 {2015-10-28 13:37:42} 3 88707 {ET TOR Known Tor Relay/Router (Not Exit) Node Traffic group 11} SRC.IP.AD.DR DST.IP.AD.DR 17 123 123 1 2522020 2376 7946 7946';
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t]: remove

File: t/unittest/detect/parser/securityonion.t [hunk 4]
```diff
@@ -0,0 +1,72 @@
+=head1 NAME
+
+example pf test
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t]: Be more explicit

File: t/unittest/detect/parser/securityonion.t [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+=cut
+
+=head1 DESCRIPTION
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t:7]: Be more explicit

File: t/unittest/detect/parser/securityonion.t [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+example pf test script
```
Comment (fdurand) [inline t/unittest/detect/parser/securityonion.t]: Be more explicit

**Review by fdurand — CHANGES_REQUESTED** [review]

**Review by tlaurion — COMMENTED** [review]
Done

**Review by fdurand — APPROVED** [review]

**Pull request merged**